ppr-ui/src/components/parties/party/PartyAutocomplete.vue

```diff
@@ -128,6 +136,13 @@ export default defineComponent({
       })
     })
 
+    const wasSelected = (val: SearchPartyIF) => {
+      if (localState.selectedCode === val.code) {
```

Comment:
Could be shortened to `localState.selectedCode === val.code ? 'wasSelected' : ''`

Reply:
shortened the if statement to ternary!